### DISCUSSION TITLE
Task timed out after 10.04 seconds on Pro plan

### DISCUSSION THREAD

**Discussion by KolbySisk** [vercel/community#5284] [Help]
### Summary

I have an API Route Handler in my App Router project that is returning an error when deployed to Vercel, but not locally. The error is: `Task timed out after 10.04 seconds`

I upgraded to the Pro plan in hopes that the increased timeout limit would fix the problem, but nothing changed.

The function succeeds about 50% of the time, which makes sense because it's hitting OpenAI and the response could potentially take longer than 10 seconds. 

I've added `export const maxDuration = 300;` to the function, which did not help.

All paths should be returning a response as well.

### Steps to Reproduce

It's a bit hard to reproduce, but here's the pseudo-code of my function:

```
// src/app/api/my-function.ts

export const maxDuration = 300;

async function handler(req: NextReques) {
  try {
    await openaiStuff();
    return new NextResponse(null, { status: 202 });
  } catch (error) {
    console.error(JSON.stringify(error));
    return NextResponse.json({ message: 'Server error!' }, { status: 500 });
  }
}

export const POST =handler;
```

**Comment by leerob** [comment]
Are you on the Pro trial or a paid Pro plan? I believe the Pro trial has the same max duration limits as the free tier (10s).

**Reply by KolbySisk** [reply]
I created a new team with a pro plan, and then moved the project over. Maybe a problem related to the project being created on a free plan?

**Comment by KolbySisk** [comment]
Figured out the problem (after a short vacation).

I have an env variable with the app's url. After migrating to a pro plan the url changed but I forgot to update the env variable. I was effectively still making calls to the old app. Updated env variable and all is good now.
